src/cli.js

```diff
@@ -236,6 +233,15 @@ Options
                 reporter.isReportServer = boolValue();
                 break;
 
+            case "report-path":
+                console.log(value.indexOf('\"'));
+                reporter.reportPath = (value.indexOf('\"') !== -1 ? value : utils.error("Invalid report-path. It must be a string don't forget double quotes around path"));
```

Comment:
on windows "\" is not getting if you miss quote

Reply:
A bit cleaner + updated error msg:
```
case 'report-path`:
    if(value.indexOf('\"') == -1) {
        utils.error("Invalid report-path. You must put double quotes around the path"));
    }
    else {
        reporter.reportPath = value;
    }
```

Reply:
Also, this MIGHT not work on MacOS, where the `""` seem to be removed automatically. Once we approve the PR, I'll have to test it on my machine prior to publish.

Reply:
I mean that MacOS might automatically remove the double-quotes from args before cli.js gets them, so the error will always occur no matter what. I might be wrong though.

Reply:
Ok i get a bug, if you run command on windows with GitBash is fail because `\` is interpret ...
On windows is mandatory to use native CMD windows.

Reply:
We can deleted check for path and its work for all OS